### PR TITLE
[COMSUP-373] EF UI limitations

### PR DESCRIPTION
**Pull request by tristanlee85**

### CODE REVIEW

File: guides/applications/v7/edge_functions.md [hunk 1]
```diff
@@ -6,34 +6,23 @@ Edge Functions enable you to execute a small piece of JavaScript code on our edg
...
 **Key information:**
 
-- This article assumes that you are familiar with our [CDN-as-Code](/applications/performance/cdn_as_code) approach for defining rules.
+- This article assumes that you are familiar with [defining rules](/applications/v7/performance/rules#rules-and-cdn-as-code) using CDN-as-Code or the {{ PORTAL }}.
 - Edge Functions requires activation. {{ ACCOUNT_UPGRADE }}
 
 {{ prereq.md }}
 
-## Defining Edge Functions {/* defining-edge-functions */}
+## Edge Functions {/* edge-function */}
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:14]:
```suggestion
## Edge Functions {/* edge-functions */}
```

File: guides/applications/v7/edge_functions.md [hunk 2]
```diff
@@ -6,34 +6,23 @@ Edge Functions enable you to execute a small piece of JavaScript code on our edg
...
-    });
-  ```
+- [CDN-as-Code](#cdn-as-code) - Define edge functions within your local project codebase.
+- [Rules UI](#rules-ui) - Define edge functions within the {{ PORTAL }}.
 
-An edge function file must export the following entry point:
+For either method, the edge function is defined as an asynchronous function exported as `handleHttpRequest` with the following signature:
 
-```js
+```js filename="./edge-functions/index.js"
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:25]: Consider splitting the intro paragraph and moving this code sample between the two halves.
like so:
... They are defined within a JavaScript file and exported as an asynchronous function named `handleHttpRequest`.
```js filename="./edge-functions/index.js"
...

Why? This code provides context to that info. Placing it under where EFs are defined breaks up the flow.
The above function is invoked when an incoming request matches a rule that ...

File: guides/applications/v7/edge_functions.md [hunk 3]
```diff
@@ -57,8 +46,42 @@ export async function handleHttpRequest(request, context) {
 }
 ```
 
+### For CDN-as-Code {/* cdn-as-code */}
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:49]:
```suggestion
### CDN-as-Code {/* cdn-as-code */}
```

File: guides/applications/v7/edge_functions.md [hunk 4]
```diff
@@ -57,8 +46,42 @@ export async function handleHttpRequest(request, context) {
...
+  export default new Router()
+    .get('/', {
+      edge_function: './edge-functions/index.js',
+    })
+    .match('/api/*', {
+      edge_function: './edge-functions/api.js',
+    });
+  ```
+
+### For Rules UI {/* rules-ui */}
```
Comment (tristanlee85) [inline guides/applications/v7/edge_functions.md:67]:
```suggestion
### Rules ({{ PORTAL }})
```

**Pull request merged**